eppo_core/src/ufc/models.rs

```diff
@@ -96,7 +96,7 @@ pub(crate) struct FlagWire {
 }
 
 /// Type of the variation.
-#[derive(Debug, Serialize, Deserialize, Clone, Copy, PartialEq, Eq)]
+#[derive(Debug, Serialize, Deserialize, Clone, Copy, PartialEq, Eq, rustler::NifUnitEnum)]
```

Comment:
@rasendubi it feels like a bit of a smell to add this but it seemed like the easiest way to make sure we can convert Elixir atoms (such as `:string` to a VariationType). Let me know if you want me to avoid adding rustler and this derive in eppo_core

Reply:
I think that's ok'ish. We have some derives for python/ruby in core as well. Unfortunately, because of rust impl rules, we can't add these implementation outside of core so I think that's fine for now.

The only thing is that this should depend on a feature flag, so we don't pull any of elixir libraries into other languages SDKs.

```suggestion
#[derive(Debug, Serialize, Deserialize, Clone, Copy, PartialEq, Eq)]
#[cfg_attr(feature = "rustler", derive(rustler::NifUnitEnum))]
```

Reply:
I couldn't get this to work yet, might need some extra eyes later.

Reply:
✨ https://github.com/Eppo-exp/eppo-multiplatform/pull/215

Reply:
Since you are modifying `eppo_core`, it will need to be published and that needs a changeset: https://github.com/Eppo-exp/eppo-multiplatform/pull/216

You can also try the tool locally with `yarn changeset`, it has a guided demo